Fixed Black movement issue.

CalderWhite opened the pull request:
Implemented a simple variable that forces the user to wait for black to move (fixed issue #24 ).

CalderWhite committed:
Attempted testing fix for issue #24

CalderWhite committed:
Patched code. Tested, works for me.

sleddog commented:
@CalderWhite can you add those generated files to the .gitignore file?
go/pkg/windows_amd64/chess.a
dist/server.exe

Also, since you seem to be running this from Windows, can you add a small section to the readme about windows development?

Thanks again!

CalderWhite committed:
Removed windows dist files and added them to .gitignore

CalderWhite commented:
@sleddog, I removed the dist files and added them to the `.gitignore`. However, I actually was a little clueless with the Windows dev, so I switched and did my development on linux. The most I did on windows was build the go source, and attempt to run it (which ended immediately).
